src/isomorphic/devtools/ReactMiscapitalizedPropNameWarningDevTool.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactMiscapitalizedPropNameWarningDevTool
+ */
+
+'use strict';
+
+var warning = require('warning');
+
+if (__DEV__) {
+
+  var warnedProperties = {};
+
+  var warnMiscapitalizedPropName = function(
+                                     componentName,
+                                     allPropNames,
+                                     missingPropName
+                                   ) {
+    var missingPropNameLowerCase = missingPropName.toLowerCase();
+    var incorrectlyCapitalizedPropName;
+
+    for (var propName of allPropNames) {
```

Comment:
Please don’t use `for of` syntax as it relies on ES6 `Symbol` being available, and we don’t polyfill it.